Manual/SQL - Basic.html

```diff
@@ -69,10 +69,10 @@
                 <h2>On the Subject of SQL - Basic</h2>
                 <p class="flavour-text">How fun is it to ride through a large table of numbers to find similarities...</p>
 
-                <p>This module is a SQL code simulator that you have to defuse by producing the proper SQL query. You have an input data in this document and the module allows you to see a goal that you must reproduce. Use the documentation to build a SQL query that will reproduce the expected solution from the input data.</p>
+                <p>This module is a SQL code simulator. To disarm it, produce a SQL query that outputs a given result. The input data is in this document, which the module displays the goal result that must be reproduced. Use the documentation to build a SQL query that will reproduce the expected result from the input data.</p>
```

Comment:
The part `, which the module displays the goal result` is very weird. I'd use: `, use it to reproduce the goal.`.